docs/Logging.md

```diff
@@ -167,4 +169,4 @@ const fastify = Fastify({
 })
 ```
 
-See https://getpino.io/#/docs/redaction for more details.
+See <https://getpino.io/#/docs/redaction> for more details.
```

Comment:
Why the `<` and `>`?

Reply:
https://github.com/DavidAnson/markdownlint/blob/v0.22.0/doc/Rules.md#md034

We can ad md034 to the ignore list if you prefer? Please advise.

Reply:
We could also:

```
See [Pino documentation](https://getpino.io/#/docs/redaction) for more details.
```

WDYT?